### PR TITLE
playerConnecting: add a lua example

### PR DESCRIPTION
**Pull request by PichotM**
Warxander asked :(

### CODE REVIEW

**Comment by warxander** [comment]
Actually, my question was about `deferrals` as I didn't use them before and, imo, the current description is not clean enough.
And I also confused about `setKickReason` because it's not used in C# example.

**Comment by PichotM** [comment]
deferrals has been implemented after setKickReason function
according to iridium, it is better to use deferrals than setKickReason callback 👍

File: content/scripting-reference/events/list/playerConnecting.md [hunk 1]
```diff
@@ -51,3 +51,23 @@ private void OnPlayerConnecting([FromSource]Player player, string playerName, dy
...
 ```
+
+##### Lua Example:
+```lua
+local function OnPlayerConnecting(name, setKickReason, deferrals)
+    local identifiers = GetPlayerIdentifiers(source)
+    deferrals.defer()
+
+    deferrals.update(string.format("Hello %s. Your steam id is being checked.", name))
+    Citizen.Wait(1000) -- not needed, we fake a loading
```
Comment (stannum-cfx) [inline content/scripting-reference/events/list/playerConnecting.md]: Why even add this delay if it's not needed. People tend to just blindly copy code snippets like these and then everyone will have a useless delay when entering the server, doesn't seem very user friendly to me.

File: content/scripting-reference/events/list/playerConnecting.md [hunk 2]
```diff
@@ -51,3 +51,23 @@ private void OnPlayerConnecting([FromSource]Player player, string playerName, dy
...
+
+    local steam = identifiers[1] -- The steam identifier is always the first identifier (if steam is enabled)
```
Comment (stannum-cfx) [inline content/scripting-reference/events/list/playerConnecting.md]: You can't just assume that it's at any specific index, if there's any identifier related changes in the future, this _could_ potentially change. It's not recommended to do it like this. Please just loop over all of them to check if _any_ identifier is the steam ID.

**Comment by PichotM** [comment]
It should be better now

**Pull request merged**